vizro-core/docs/pages/user-guides/custom-tables.md

```diff
@@ -1,15 +1,17 @@
-# How to create custom tables
+# How to create custom Dash AG Grids and Dash DataTables
 
-If you want to use the [`Table`][vizro.models.Table] model to and to create a custom [table](table.md) you can create your own custom table, e.g. when requiring computations that can be controlled by parameters.
+In case that the available arguments for the [`AgGrid`][vizro.models.AgGrid] or [`Table`][vizro.models.Table] models are not sufficient,
+there is always the possibility to create a custom Dash AG Grid or Dash DataTable.
+
+One reason could be that you want to create a table/grid that requires computations that can be controlled by parameters (see below example).
 
 For this, similar to how one would create a [custom chart](../user-guides/custom-charts.md), simply do the following:
 
-- define a function that returns a  `dash_table.DataTable` object
-- decorate it with the `@capture("table")` decorator
+- define a function that returns a `dash_ag_grid.AgGrid` or `dash_table.DataTable` object
```

Comment:
```suggestion
- Define a function that returns a `dash_ag_grid.AgGrid` or `dash_table.DataTable` object.
```

Reply:
I have simplified the Kedro docs guide for Vizro docs when it comes to bullets. It would be ideal to have lower case start and no period (when not a full sentence) but then there are cases where bullets contain two sentences, and it all gets inconsistent. So I've suggested all bullets start with a capital and end with a period. Where this isn't the case already in docs, I'll fix as I go through or we can do them as we update docs for other reeasons.